### DISCUSSION TITLE
Configuration for trunk based development

### DISCUSSION THREAD

**Discussion by mowies** [semantic-release/semantic-release#2041] [Configuration] [answered]
I am trying to introduce semantic-release in my project, but I want to follow a different branching strategy closer to trunk based development. I am trying to configure the branches so that `main` is in prerelease mode (easy enough) but I want a separate release branch per minor version (eg `1.0`, `1.1`) where the initial release for the branch is already over there and not on `main`.
Is this somehow possible? I found a few [issue threads](https://github.com/semantic-release/semantic-release/issues/1529) where people talked about trunk based development but there were no concrete answers anywhere.

**Comment by travi** [comment, marked as answer]
thank you for the nudge. i've been meaning to respond to that thread and just did. lets keep the conversation there, rather than starting another conversation here.
